media/css/submitFeedback.css

```diff
@@ -6,20 +6,37 @@ input[type='submit'] {
     cursor: pointer;
 }
 
+select {
+    background-color: var(--vscode-settings-selection-background);
+    color: var(--vscode-settings-textInputForeground);
+    width: 80%;
```

Comment:
it's been a while, but any reason why 80% and not something like `100vw` (i.e. based on container width vs viewport width)?